contracts/WildcardSteward_v2.sol

```diff
@@ -587,33 +600,30 @@ contract WildcardSteward_v2 is Initializable {
 
     function withdrawBenefactorFundsTo(address payable benefactor) public {
         require(
-            benefactorLastTimeCollected[benefactor].add(1 days) > now,
+            // QUESTION? Should this 1 day throttle limit be configurable?
```

Comment:
@moose-code thoughts?

Reply:
I saw in your other PR you had a function to change this limit. I commented that there should be a require not allowing the limit to be set to high maybe.

What limit are we thinking first off?

How did you come up with 1 day? I think it makes sense but toying with 7 days etc since withdrawing won't be a common event

Reply:
I've made it configurable (it also just makes it easier to test when this is configurable).

No big logic behind 1 day, it is long enough for us to get a notification from our monitoring software to say there is a problem (if there is a problem) and solve it.

1 hour for example might not be enough. 7 days seems unnecessarily long.